recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - setuptools
     - wheel
   run:
+    - fsspec
```

Comment:
Looks like fsspec was dropped again in https://github.com/Lightning-AI/litdata/pull/380, so we'll skip this v0.2.27 release and jump to v0.2.28 directly in https://github.com/conda-forge/litdata-feedstock/pull/29